backend/models/order.js

```diff
@@ -17,6 +21,10 @@ module.exports = (sequelize, DataTypes) => {
       shortId: DataTypes.STRING,
       // Current status of the order.
       paymentStatus: DataTypes.ENUM(OrderPaymentStatuses),
+      // Type of payment made
+      paymentType: isPostgres
```

Comment:
Tests were failing on SQLite (during DB migration). "Alter type" isn't supported I think. I could just return `Promise.resolve()` in the migration if SQLite though (since the Enum won't require updating when running all in sequence),